bzt/modules/influxdb_reporter.py

```diff
@@ -0,0 +1,378 @@
+"""
```

Comment:
how do we test the new module?

Reply:
Hello, 

You mean e2e test ? So you need to setup influxdb 1.x db, create a taurus file with this module properly configured. To visaulize the result a graphana instance with a custom dashboard can be helpful. I can provide a sample.

For a unit test it's a bit hard (I looked about the others reporters but no tests found) but some idea are welcomed if you have.